tests/src/test/scala/pathy/path.scala

```diff
@@ -100,133 +84,194 @@ class PathSpecs extends Specification with ScalaCheck {
     ).toList must_== List("c", "foo", "bar", "flat.md")
   }
 
-  "parseRelFile - image.png" in {
-    parseRelFile("image.png") must beSome(file("image.png"))
-  }
+  "parseRelFile" >> {
```

Comment:
Ya, I take back my comment on a `Arbitrary[PathCoded]`. I think the tests are mostly meant to test `placeholder` and `/` is easier than `\\`.